tests/integrationv2/tox.ini

```diff
@@ -17,12 +19,14 @@ deps =
 # -rpfsq : print a (r)eport with (p)assed tests, (f)ailed tests, and (s)kipped tests
 # --durations=10 : print the 10 slowest tests
 commands =
+    bash -c 'export TEST_NAME=$(basename $TOX_TEST_NAME | sed ''s/test_//''| sed ''s/.py//''); \
```

Comment:
I think this is okay because `pytest` is the last command that's executed, so if it errors this will still error? But should we `set -e` in case we add any additional commands here? Just worried that we'll accidentally start ignoring the `pytest` exit code.